po/fa.po

```diff
@@ -1024,26 +1024,26 @@ msgstr "همزمانی: عصر"
 
 #: src/index.md:3
 msgid ""
-"[![Build workflow](https://img.shields.io/github/actions/workflow/status/"
-"google/comprehensive-rust/build.yml?style=flat-square)](https://github.com/"
-"google/comprehensive-rust/actions/workflows/build.yml?query=branch%3Amain)"
+"[![Build workflow](https://img.shields.io/github/actions/workflow/status/google/"
+"comprehensive-rust/build.yml?style=flat-square)](https://github.com/google/"
+"comprehensive-rust/actions/workflows/build.yml?query=branch%3Amain)"
```

Comment:
This is the kind of change that should not happen. If you install `dprint` and the Gettext tools, you can run `dprint fmt` to fix it.